CHANGELOG.md

```diff
@@ -1,8 +1,15 @@
 # Changelog
 
-## Currently in development
 
-* New maintenance task: update missing checksum one, after dss start.
+## 1.5.0 2020-11-03
+
+* New maintenance task: update missing checksum once, after dss starts.
```

Comment:
```suggestion
## 1.6.0 2021-01-22

* Fix for workflow result registration: fetch sample by identifier instead of search for robustness against indexing problems
* Retry sample tracking updates twice and log failures that occur

## 1.5.0 2020-11-03

* New maintenance task: update missing checksum once, after dss starts.
```